.github/workflows/update-headlamp-version.yml

```diff
@@ -6,7 +6,7 @@ on:
     - cron: "0 10 * * 1"
 env:
   GOPROXY: https://proxy.golang.org
-  GO_VERSION: '1.23.4'
+  GO_VERSION: '1.24.0'
```

Comment:
please update the source of the issue, which is in the go-updater
check the script that is called by 
```
make update-golang-version
```

that one is missing this file to update